src/tables.ts

```diff
@@ -146,14 +141,12 @@ export const CommonMetrics: Record<string, any> = {
   },
 
   TabSlide: {
-    fontFamily: 'Times New Roman, serif',
```

Comment:
So our previous VexFlow contributors have decided that some elements should be `Arial, sans-serif` but other elements should be `Times New Roman, serif`..... By removing all these, does that mean all of the elements will use `Roboto Slab`? (Or Academico, when we switch over.)

Reply:
Should it be easy to customize? Or at least we can document how to customize it by adding such a fontFamily entry inside the CommonMetrics object.

Reply:
This is right.